conda/gateways/logging.py

```diff
@@ -91,4 +91,3 @@ def trace(self, message, *args, **kwargs):
 
 logging.addLevelName(TRACE, "TRACE")
 logging.Logger.trace = trace
-initialize_logging()
```

Comment:
Even though the `@memoize` decorator is on this function, it's still getting called twice, like python is creating two different instances of it.  Don't really understand it.  But we also don't need to call it here.